istioctl/pkg/util/constants.go

```diff
@@ -15,9 +15,6 @@
 package util
 
 const (
-	// DefaultProxyAdminPort is the default port for the proxy admin server
```

Comment:
looks like we should reuse this one everywhere, `util` should be safer to import.

Reply:
updated, thx

Reply:
@zirain are our linter rules working as expected? I thought https://github.com/istio/istio/pull/57240/files#diff-e216033b597b5fedadbec7b0e9c9c62b2314922a169332a9df677ff47772dccaR34 would trigger the dep guard due to istioctl's import restrictions, but i'm not 100% sure.

Reply:
I've checked this, and there are packages importing `istioctl` elsewhere in `tools/bug-report`, like: https://github.com/istio/istio/blob/master/tools/bug-report/pkg/bugreport/bugreport.go#L32-L33. 
I understand that shouldn't be a problem?